ckanext/ioos_theme/templates/feedback/form.html

```diff
@@ -14,19 +14,17 @@
          be simplified? #}
       <h1 class="page-heading">{{ _('Submit Feedback')}}{% if c.package_name is not none and c.package_name != '' %} for Dataset ID {{c.package_name}}{% endif %}</h1>
       {% block form %}
-        {% if c.package_name is not none and c.package_name != '' %}
-          {% set action_loc = h.url_for(controller='package', action='read', id=c.package_name) %}
-        {% else %}
-          {% set action_loc = "" %}
-        {% endif %}
+
+      {% set action_loc = "" %}
```

Comment:
We could remove this entirely and just set the action to ""

Reply:
Good point. I'll do that. ty.